util/predicates/cluster_predicates.go

```diff
@@ -167,6 +168,45 @@ func ClusterUnpaused(logger logr.Logger) predicate.Funcs {
 	return Any(log, ClusterCreateNotPaused(log), ClusterUpdateUnpaused(log))
 }
 
+// ClusterControlPlaneInitialized returns a Predicate that returns true on Update events
+// when cluster gets ControlPlaneInitializedCondition set to true.
```

Comment:
```suggestion
// when ControlPlaneInitializedCondition on a Cluster changes to true.
```

nit

Since the predicate only cares about the cases where the Cluster's ControlPlaneInitializedCondition transitions to true (from false or unset) it would be good to call that out in the comment.